src/Maestro/DependencyUpdater/DependencyUpdater.cs

```diff
@@ -168,12 +168,12 @@ select new
 
                 var subscriptionsAndBuilds = await subscriptionsToUpdate.ToListAsync(cancellationToken);
                 Logger.LogInformation($"Will update '{subscriptionsAndBuilds.Count}' subscriptions");
+
                 foreach (var s in subscriptionsAndBuilds)
                 {
                     Logger.LogInformation($"Will update {s.subscription} to build {s.latestBuild}");
+                    await UpdateSubscriptionAsync(s.subscription, s.latestBuild);
```

Comment:
This should have a try/catch so one failure doesn't cause all the following subscriptions to be skipped.

Reply:
@alexperovich I think this actually ends up in ActionRunner, which has a try catch.

Reply:
I see it does, the only exceptions that can get here are a problem with Logger.BeginScope, its probably fine then.